### PR TITLE
Create text proposal

### PR DESCRIPTION
**Pull request by akhilesh-singh-Lab49**
governor text proposal added.

### CODE REVIEW

File: deployment/governorTextProposal.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import { Deployment } from "./deployContractOnTestnet";
+
+dotenv.config();
+
+async function main() {
+  const deployment = new Deployment();
+  const filePath = "./artifacts/contracts/common/GovernorTextProposal.sol/GovernorTextProposal.json";
+  console.log(`Deploying Governor Text Proposal contract...`);
+  const deployedContract = await deployment.deployContractAsClient(filePath, new ContractFunctionParameters());
+  console.log(`BaseHTS deployed ${JSON.stringify(deployedContract)}`);
```
Comment (amandeeprehal-lab49) [inline deployment/governorTextProposal.ts:14]: pls correct the `BaseHTS deployed` log text

File: integrationTest/GovernorMethods.ts [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+  TokenId,
+  AccountId
+} from "@hashgraph/sdk";
+
+import ClientManagement from "./utils/utils";
+import { ContractService } from "../deployment/service/ContractService";
+
+  import dotenv from "dotenv";
+  
+  dotenv.config();
```
Comment (amandeeprehal-lab49) [inline integrationTest/GovernorMethods.ts:19]: correct the indentation

File: deployment/governorTextProposal.ts (hunk 1 above)
Comment (akhilesh-singh-Lab49) [reply deployment/governorTextProposal.ts:14]: Done

File: integrationTest/GovernorMethods.ts (hunk 2 above)
Comment (akhilesh-singh-Lab49) [reply integrationTest/GovernorMethods.ts]: done

File: integrationTest/GovernorMethods.ts [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+let client = clientManagement.createOperatorClient();
+
+const treasurerClient = clientManagement.createClient();
+const { treasureId, treasureKey } = clientManagement.getTreasure();
+
+const htsServiceAddress = contractService.getContract(
+  contractService.baseContractName
+).address;
+
+const readFileContent = (filePath: string) => {
```
Comment (amandeeprehal-lab49) [inline integrationTest/GovernorMethods.ts:33]: are you using this code?

File: integrationTest/GovernorMethods.ts [hunk 4]
```diff
@@ -0,0 +1,169 @@
+import { BigNumber } from "bignumber.js";
+import * as fs from "fs";
+import {
+  ContractExecuteTransaction,
+  ContractFunctionParameters,
+  Hbar,
+  TokenCreateTransaction,
+  TokenType,
+  TokenSupplyType,
+  TokenId,
```
Comment (amandeeprehal-lab49) [inline integrationTest/GovernorMethods.ts:10]: Pls remove unused code.

Comment (akhilesh-singh-Lab49) [reply integrationTest/GovernorMethods.ts]: done

File: integrationTest/GovernorMethods.ts (hunk 3 above)
Comment (akhilesh-singh-Lab49) [reply integrationTest/GovernorMethods.ts]: done

**Review by amandeeprehal-lab49 — APPROVED** [review]

**Pull request merged**